tests/apis.js

```diff
@@ -52,80 +51,300 @@ test.cb.after.always(t => {
     });
 });
 
+/**
+ * Tests attributes with references
+ * @param  {object} t - ava testing
+ * @param  {object|array} attrs object containing attributes to test
+ */
+const referencerTests = (t, attrs) => {
```

Comment:
I'd like some more comments in here explaining what each block is doing so we don't have to divine it from the code each time